src/Init/Data/BitVec/Basic.lean

```diff
@@ -170,6 +170,9 @@ instance : GetElem (BitVec w) Nat Bool fun _ i => i < w where
 @[simp] theorem getLsb?_eq_getElem? (x : BitVec w) (i : Nat) :
     x.getLsb? i = x[i]? := rfl
 
+theorem getElem_eq_toNat_testBit (x : BitVec w) (i : Fin w) :
+  x[i] = x.toNat.testBit i := rfl
```

Comment:
This should use separate `i : Nat` and `h : i < w` arguments, otherwise it's only applicable for `getElem` of the coercion of a Fin.